### DISCUSSION TITLE
Wheathermap 1.0 not showing Group

### DISCUSSION THREAD

**Discussion by Nam1681** [thurban/plugin_weathermap#113] [Q&A]
Hello,

After install Weathermap 1.0, I noticed Group MAP not showing anymore for Page Style = Full Images. Please help. Thanks.

**Comment by TheWitness** [comment]
Can you attach some screen shots before and after to explain?

**Reply by Nam1681** [reply]
This setting from 0.98a and show Group Map.
![12](https://user-images.githubusercontent.com/87417741/229496434-4b9b77b9-0da1-4064-83db-ad9f1bfadb82.PNG)
![13](https://user-images.githubusercontent.com/87417741/229496456-bd851b68-9fc2-480d-b522-0d42dcafd6fc.PNG)

And this from 1.0, it not showing Group of Map. 
![14](https://user-images.githubusercontent.com/87417741/229496676-1cac6e54-fb6a-4092-9fc5-f9074aa850df.PNG)
![15](https://user-images.githubusercontent.com/87417741/229496690-5ed36ebd-82fd-4f89-ab30-c0144bf9f28c.PNG)

**Reply by TheWitness** [reply]
Zip up your Weathermap plugin sans the config and output folders and attach to this ticket.  You have a slightly modified version of the plugin.  No promises as I'm on the 1.3.x release right now, but will take a cursory look.
